HISTORY.md

```diff
@@ -31,6 +31,7 @@
 * Added is_full_compaction to CompactionJobStats, so that the information is available through the EventListener interface.
 * Add more stats for MultiGet in Histogram to get number of data blocks, index blocks, filter blocks and sst files read from file system per level.
 * SST files have a new table property called db_host_id, which is set to the hostname by default. A new option in DBOptions, db_host_id, allows the property value to be overridden with a user specified string, or disable it completely by making the option string empty.
+* Methods to create customizable extensions -- such as TableFactory -- are exposed directly through the Customizable base class (from which these objects inherit).  This change will allow these Customizable classes to be loaded and configured in a standard way (via CreateFromString).  More information on how to write and use Customizable classes is in the header file and on the Wiki.
```

Comment:
I think we should mention virtual Configurable* Inner() is removed from configurable.h protected functions in "Public API Change". And if any user that are using it, how should they do.

Reply:
The Inner was introduced when the Configurable class was introduced.  Since this is a new class as of the last release and is used by RocksDB extension developers (without the benefit of the Wiki documentation), I would be amazed if anyone is using this currently.  If you feel strongly it should be documented, I will but I think it is highly unlikely to be used and more likely to cause confusion.

Reply:
Removed the Inner altogether for now as it is not currently needed.  Will re-introduce it later if required.